libs/novui/package.json

```diff
@@ -129,13 +129,22 @@
     }
   },
   "dependencies": {
-    "@mantine/code-highlight": "^7.10.2",
-    "@mantine/core": "^7.10.0",
-    "@mantine/hooks": "^7.10.0",
+    "@mantine/code-highlight": "^7.11.2",
+    "@mantine/core": "^7.11.2",
+    "@mantine/hooks": "^7.11.2",
+    "@mantine/tiptap": "^7.11.2",
```

Comment:
@ainouzgali FYI I updated all the Mantine packages to have the same version since it can cause issues if they get mismatched!